src/modules/Dashboard/features/Applet/Schedule/EventForm/EventForm.types.ts

```diff
@@ -85,6 +87,7 @@ export type GetEventFromTabs = {
   hasNotificationsErrors?: boolean;
   hasAlwaysAvailableOption?: boolean;
   'data-testid'?: string;
+  removeWarnings?: Warning;
```

Comment:
Just a suggestion to rename the `Warning` type as well, following on the heels of your `removeWarning` ⇒ `removeWarnings` change:
```suggestion
  removeWarnings?: EventFormWarnings;
```